src/platformdirs/macos.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from .api import PlatformDirsABC
+
+
+class MacOS(PlatformDirsABC):
+    """
+    Platform directories for the macOS operating system. Follows the guidance from `Apple documentation
+    <https://developer.apple.com/library/archive/documentation/FileManagement/Conceptual/FileSystemProgrammingGuide/MacOSXDirectories/MacOSXDirectories.html>`_.
```

Comment:
Can you move the URL to the end of the parragraph as done [here](https://raw.githubusercontent.com/platformdirs/platformdirs/main/docs/index.rst)?

This helps keep the docstring more readable in it's raw format.

Reply:
I disagree in this instance, I prefer to keep links inline 👍